doc/modules/feature_selection.rst

```diff
@@ -267,9 +283,6 @@ of features non zero.
      Journal of the Royal Statistical Society, 72 (2010)
      http://arxiv.org/pdf/0809.2932
 
-   * F. Bach, "Model-Consistent Sparse Estimation through the Bootstrap"
-     http://hal.inria.fr/hal-00354771/
```

Comment:
That's good point, now I think credit to Bach's Bolasso need to be included.  What about keeping the reference and making a little note clarifying that what is implemented is the Stability procedure, and that a similar randomization approach can be obtained via the Bolasso?